docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md

```diff
@@ -7,7 +7,7 @@ import TabItem from '@theme/TabItem';
 import Congratulations from '../guides/connecting_to_your_data/components/congratulations.md'
 
 This guide will help you integrate Great Expectations (GE) with [Google Cloud Platform](https://cloud.google.com/gcp) (GCP) using our recommended workflow.
-    
+
```

Comment:
There's a lot of misc whitespace changes in this PR - my editor automatically removes any extraneous whitespace on save so please ignore.